README.md

```diff
@@ -29,6 +29,7 @@ A boilerplate-free Scala library for loading configuration files.
 - [Handling missing keys](docs/handling-missing-keys.md)
 - [Example](docs/example.md)
 - [Whence the config files](docs/whence-the-config-files.md)
+- [Playing nice with Spark's Shapeless](docs/playing-nice-with-sparks-shapeless.md)
```

Comment:
This should also be added to `core/src/main/tut/README.md` or it will be overwritten when the docs are next generated.